charts/argo-cd/values.yaml

```diff
@@ -606,7 +620,8 @@ redis:
   podLabels: {}
```

Comment:
REDIS is going to need a region affinity as well. It is also going to need resource requests / limits if that is not supplied by default.

There is actually not a great use case to deploy ArgoCD in different regions now that I think of it. The control plane is in ORD1 so that's really where we want people to deploy it. If it makes your life easier to just statically configure everything for ORD1, do that.

Reply:
That makes sense, then I will statically configure things for ord1.